src/doc/style-guide/src/expressions.md

```diff
@@ -514,10 +541,10 @@ if let ... {
 }
 ```
 
-If the control line needs to be broken, then prefer to break before the `=` in
-`* let` expressions and before `in` in a `for` expression; the following line
-should be block indented. If the control line is broken for any reason, then the
-opening brace should be on its own line and not indented. Examples:
+If the control line needs to be broken, prefer to break before the `=` in `*
```

Comment:
Similar thought as earlier wrt to "prefer", but don't feel strongly

```suggestion
If the control line needs to be broken, break before the `=` in `*
```

Reply:
Here, I *think* that this "prefer" represents "here's how to choose among multiple possible sources of line breaks". For instance, if you *have* to break it in another place because this break isn't sufficient, and with it broken in that other place you don't need to break before the `=`, then you shouldn't actually break it before the `=`.

If we want to say "always break before the `=` if you have to break at all, even if when done you don't need that break to stay under the line width", we could make this change, but I don't think that's accurate here?